### ISSUE TITLE
[feature request] have clipboard auto sorting be optional

### ISSUE THREAD

**Issue by kokopossum** [futo-org/android-keyboard#355]
have an option for the user to disable the clipboard sorting recently pasted/used to to the top of the board. I just find it annoying but im probably in the minority.

**Comment by FireNinja7365** [comment]
I agree, it does get annoying

**Comment by znbaboy** [comment]
i agree. its annoying. the default behavior should be *last copied thing to the top*. It should not change positions by *last pasted item*

Gboard does it very well

**Comment by UnitedShoes** [comment]
I would love the option to disable Clipboard History, but still use the pinned/saved clipboard items. I have little use for clipboard history, but when it comes to quickly pasting a string for a game I play, clipboard is  awesome. 

I really hope this gets reworked a little, since pinning the pastes doesn't do much more than prevent it from being auto-deleted. My random clipboard history always pushes my shortcut pastes, making it really hard to get muscle memory.

**Comment by TeoMorabito** [comment]
And pinned item should be at the top below the 2 last copy's
Right now I have to scroll down quite far to get pinned clipboard items.

Important also, once you click (paste) a clipboard item, the clipboard menu should close and go back to the keyboard

I rarely see a point to paste 2 seperate items consecutively.
